_posts/events/2025-04-22-community-call.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: single
+title: "pyOpenSci Community Call 5 May 2025 @ 10 am Mountain Time: Right Time, Right Community: My OSS Journey"
+excerpt: "We’re kicking off a special contribute to open source with a special Open Source Voices session! Pavritha will share her open source journey and what helped her grow in the open source world."
```

Comment:
a special contribute ?? Seems like something may be missing

Reply:
eeks - fixed! good catch!!